src/Vault.sol

```diff
@@ -288,14 +303,13 @@ contract Vault {
      * collateralised assets can fluctuate, the haircut guarantees that the vault
      * remains over-collateralised with a high confidence level (99,9%+). The size of the
      * haircut depends on the underlying risk of the assets in the vault, the bigger the volatility
-     * or the smaller the on-chain liquidity, the biggert the haircut will be.
+     * or the smaller the on-chain liquidity, the bigger the haircut will be.
      */
     function getCollateralValue() public view returns (uint256 collateralValue) {
```

Comment:
This function should not in real time calculate again the collateralFactor, it should use the stored collateral factor